nautobot/dcim/models/devices.py

```diff
@@ -1752,4 +1803,4 @@ def get_children(self):
         """
         Return the set of child Modules installed in ModuleBays within this Module.
         """
-        return Module.objects.filter(parent_module_bay__parent_module=self)
+        return Module.objects.filter(parent_module_bay__parent_module=self.pk)
```

Comment:
Why this change?

Reply:
I might have chosen the wrong side on a merge conflict. I will change back.